demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,233 @@
+# Image classification models fixed-point quantization tutorial
+
+> Please install Paddle 1.7.1 or higher version and PaddleSlim
+
+## Overview
+
+Quantization is an important method for model compression. In PaddlePaddle, the quantization strategy `post` is the model obtained by using offline quantization, and` aware` is the model obtained by online quantization training. This tutorial introduces the use of training quantization strategy (`aware`) to quantify the image classification model and accelerate through MKL-DNN optimization. On the Intel (R) Xeon (R) Gold 6271 machine, after quantization and MKL-DNN acceleration, the performance of the INT8 model on a single thread is 3 to 4 times faster than original FP32 model. At present, the quantizable ops include `conv2d`, `depthwise_conv2d`, `mul`, `matmul`; Meanwhile, in the MKL-DNN optimization stage, we will fuse many other ops, including `batch_norm`, `relu`, `brelu`, `elementwise_add`, etc. After quantization and fuses, INT8 models performance will be greatly improved. For details about MKL-DNN optimization users can refer to [SLIM QAT for INT8 MKL-DNN] (https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md)
```

Comment:
These are my suggestions of corrections. "..." means unchanged part.
"... for model compression and inference performance improvement. PaddlePaddle supports two quantization strategies: `post` and `aware`. In the `post` strategy a trained model is quantized. In the `aware` strategy a model is trained for quantization and then quantized. This tutorial presents the use of the `aware` training quantization strategy to quantize an image classification model and accelerate it through DNNL optimizations. On Intel (R) Cascade Lake class CPU machines, 8-bits quantization, graph optimizations and DNNL acceleration yields performance of a quantized model up to 4 times better than of an original FP32 model. Currently, quantizable operators ... `matmul`.  For details ..."

Reply:
Ok